rh-tictactoe/src/redux/topbar/actions.js

```diff
@@ -0,0 +1,14 @@
+import { cleanInfo } from '../../services/userService';
+
+export const actions = {
+  LOG_OUT: 'LOG_OUT'
+};
+
+export const actionCreator = {
+  logOut: dispatch => {
```

Comment:
i would not create another section in redux for the topbar, just leave it as it was before